src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
Encapsulation of this method currently would make every supported browser return `true` on this support test in `test/unit/support.js` where run outside of an iframe except on TestSwarm where it would return `false` in IE11.

Also, if we want to consider what @dmethvin said:
> I wonder if we need to handle the case where a jQuery in one frame is manipulating elements in another frame. 

then it would complicate our TestSwarm situation even further.

I feel changing this one property in a way you propose is complicated enough on its own from the testing point that it warrants a separate issue, we shouldn't require the OP to deal with all that.